controllers/utils/acm.go

```diff
@@ -0,0 +1,75 @@
+package utils
```

Comment:
Rename the file to "managedclusterview.go" and add only functions related to it.
Having generic files usually makes it a dumping ground for everything.

Reply:
done